src/utils/collectibles.js

```diff
@@ -58,3 +59,12 @@ export function findCollectible(
     ({ contractAddress, id }) => addressesEqual(contractAddress, contractAddressToFind) && id === idToFind,
   );
 }
+
+export function isChainIcon(nameOrUrl: ?string): boolean {
+  if (!nameOrUrl) return false;
+
+  const chainInfo = nativeAssetPerChain[nameOrUrl];
+  if (!chainInfo) return false;
+
```

Comment:
- Could we please move this out of this file and into a utils file for tokens? This file is specific to NFTs
- Could we add an error log here to state what name or URL was not found so we can keep track of URL problems for tokens